modules/core/04-channel/keeper/keeper.go

```diff
@@ -827,28 +827,28 @@ func (k *Keeper) PruneAcknowledgements(ctx context.Context, portID, channelID st
 	return totalPruned, totalRemaining, nil
 }
 
-// GetV2Counterparty returns a version 2 counterparty for the given port and channel ID
-// by converting the channel into a version 2 counterparty
-func (k *Keeper) GetV2Counterparty(ctx context.Context, portID, channelID string) (packetserver.Counterparty, bool) {
+// GetV2Channel returns a version 2 channel for the given port and channel ID
+// by converting the v1 channel into a version 2 channel
+func (k *Keeper) GetV2Channel(ctx context.Context, portID, channelID string) (packetserver.Channel, bool) {
```

Comment:
Can probably remove this from classic at some point later and have it fully in the v2 pkg/codepath.